descwl_coadd/coadd.py

```diff
@@ -1331,202 +793,85 @@ def _finish_init(self):
             store_pixels=False,
         )
 
-
-def make_stack_psf(psf_image):
-    """
-    make fixed image psf for stack usage
-    """
-    return KernelPsf(
-        FixedKernel(
-            afw_image.ImageD(psf_image.astype(np.float))
+        flags_for_maskfrac = self.coadd_exp.mask.getPlaneBitMask('BRIGHT')
+        self.meta['mask_frac'] = get_masked_frac(
+            mask=self.ormask,
+            flags=flags_for_maskfrac,
         )
-    )
 
 
-def make_stack_wcs(wcs):
+def get_coadd_psf_wcs(coadd_wcs, coadd_bbox, psf_dims):
```

Comment:
This routine appears wrong for the same reason as before. The coadd PSF WCS is the coadd WCS but with the PSF stamp labeled with the right bounds.

Reply:
no longer used, will remove